app/assets/styles/bulletin.css

```diff
@@ -176,6 +175,10 @@ hr.image {
 	padding: 1.5em;
 }
 
+/* .article-content {
+	display: flex;
+} */
```

Comment:
Please get in the habit of reviewing your PRs to make sure you remove simple errors, such as this. We don't allow code that is commented out into the repo.

Reply:
This is not an error.  I am trying to understand why the grid items in this grid are not behaving as expected when displayed flex.

Reply:
Okay. But the code has no value to the final PR. Please remove it.

Reply:
I found the answer and made the corresponding change.